functionary/ui/tables/schedule_task.py

```diff
@@ -0,0 +1,44 @@
+import django_tables2 as tables
+from django.urls import reverse
+from django.utils.html import format_html
+
+from core.models.scheduled_task import ScheduledTask
+
+
+class ScheduledTaskTable(tables.Table):
+    name = tables.Column(
+        linkify=lambda record: reverse(
+            "ui:scheduledtask-detail", kwargs={"pk": record.id}
+        ),
+    )
+    function = tables.Column(
+        linkify=lambda record: reverse(
+            "ui:function-detail", kwargs={"pk": record.function.id}
+        ),
+    )
+    last_run = tables.DateTimeColumn(
+        accessor="most_recent_task__created_at",
+        verbose_name="Last Run",
+        linkify=lambda record: reverse(
```

Comment:
This throws an exception if the task hasn't run yet.  Instead of using a lambda here, you should write a function (in this same file, above the class) that you point this to instead.  The function would return either this reverse value if `record.most_recent_task` is set, or `None`.